Week06/lesson12/homework/rock_paper_scissors_starter/js/main.js

```diff
@@ -0,0 +1,27 @@
+// storing the running totals, starting at 0
+var humanScore = 0;
+var computerScore = 0;
+
+// function human clicks rock, scores dont change
+function rock() {
+  $("rock").click;
+  $("status").innerHTML = "The bot played Rock, you draw!";
```

Comment:
You've forgotten the `#` in all your `$("idNameHere")` bits of code in this file, it should be:

```
$("#idNameHere") // don't forget the #
```